src/Geta.Epi.MediaReferenceSelector/web.config.install.xdt

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+  <episerver.shell xdt:Transform="InsertIfMissing">
+    <protectedModules rootPath="~/EPiServer/" xdt:Transform="InsertIfMissing">
```

Comment:
The path will be custom and different from project to projecct. Many use /Episerver, but some use /Epi or /cms